electron/legendary/library.ts

```diff
@@ -550,4 +545,87 @@ class LegendaryLibrary {
   }
 }
 
-export { LegendaryLibrary }
+/**
+ * Runs legendary with the given command
+ * @param commandParts The command to run, e. g. 'list', 'egl-sync'...
+ */
+// TODO: This isn't the right place for this, but I don't know where else to put it. 'launcher.ts' maybe?
+export async function runLegendaryCommand(
```

Comment:
Pretty good refactor, this avoid repetition by a lot 👍🏽

Reply:
Yes, pretty good especially when we add a `Heroic Runner` in the future and also others like Amazon and so on.
We can follow the same pattern and will be a lot easier to grow from here :)